addons/mass_mailing/views/snippets/s_comparisons.xml

```diff
@@ -8,16 +8,16 @@
                 <div class="col-lg-6 s_col_no_bgcolor pt32 pb16">
                     <div class="card o_cc o_cc2">
                         <div class="card-header"><h2 style="text-align: center;"><span style="font-weight: bolder;">DEFAULT</span></h2></div>
-                        <div class="bg-white pt24 pb24">
+                        <div class="bg-white pt24 pb24" style="text-align: center;">
```

Comment:
When applying new style, it is recommended to use bootstrap as much as possible, here I think `text-center` would work fine?

Reply:
(Small note: if this content is meant to be sent by email, sometimes we don't want to use bootstrap, as it will not work inside the email content itself. Not sure if that applies here, to double-check).

Reply:
It would apply here, yes, but the actual reason for this choice is that the rest of the document used `style="text-align: center;"`. However, I see here that the above div uses the Card class, and its containing div uses col-lg-6 classes, which _are_ bootstrap classes, so.... maybe it would work?